react/components/ProductSummaryPrice.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react'
+import classNames from 'classnames'
+import ProductPrice from 'vtex.store-components/ProductPrice'
+import { path } from 'ramda'
+import productSummary from '../productSummary.css'
+import { Spinner } from 'vtex.styleguide'
+
+export default class ProductSummaryPrice extends Component {
+  commertialOffer = () => path(['sku', 'seller', 'commertialOffer'], this.props.product)
```

Comment:
Is there any specific reason for it to be a function?

Reply:
not really, will change